editor.planx.uk/src/pages/FlowEditor/lib/analyticsProvider.tsx

```diff
@@ -10,16 +10,20 @@ export type AnalyticsType = "init" | "resume";
 type AnalyticsLogDirection = AnalyticsType | "forwards" | "backwards";
 
 export type HelpClickMetadata = Record<string, string>;
+export type NextStepsLinkMetadata = Record<string, string>;
```

Comment:
What could tightening this type up look like? Would we explicitly expect a key of 'url' and a string which would be the value? I'm no too sure how I would improve this, it's been too long since I've worked with types 😂

Reply:
As discussed in a call this was resolved with `Record<'selectedUrls', string[]` i.e. more specific key.